central/search/service/service_impl_test.go

```diff
@@ -244,14 +250,25 @@ func (s *SearchOperationsTestSuite) TestAutocompleteForEnums() {
 	ctx := sac.WithGlobalAccessScopeChecker(context.Background(), sac.AllowAllAccessScopeChecker())
 
 	// Create Policy Searcher
-	policyStore := policyStoreMocks.NewMockStore(s.mockCtrl)
-	policyStore.EXPECT().GetAll(gomock.Any())
-	idx, err := globalindex.MemOnlyIndex()
-	s.NoError(err)
-	policyIndexer := policyIndex.New(idx)
-	s.NoError(policyIndexer.AddPolicy(fixtures.GetPolicy()))
-	policySearcher := policySearcher.New(policyStore, policyIndexer)
-	ds := policyDatastore.New(policyStore, policyIndexer, policySearcher, nil, nil)
+	var policyIndexer policyIndex.Indexer
+	var ds policyDatastore.DataStore
+	if env.PostgresDatastoreEnabled.BooleanSetting() {
+		policyStore := policyPostgres.New(s.pool)
+		policyIndexer = policyPostgres.NewIndexer(s.pool)
+		s.NoError(policyIndexer.AddPolicy(fixtures.GetPolicy()))
+		s.NoError(policyStore.Upsert(ctx, fixtures.GetPolicy()))
```

Comment:
Not related to your PR but inserting directly into store is an upsert pattern. (My guess is it is done this way likely to skip some steps in ds)